### PR TITLE
Feature/5112 display primary contact form

### PR DESCRIPTION
**Pull request by alibombaye**
Story [AB#5112](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/5112)
Task [AB#7109](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7109)

Screenshot 2020-05-28 at 09 48 45

### CODE REVIEW

**Comment by kat1906** [comment]
you should definitely extract this out as it is now in 2 places

File: app/pages/sections/supplier/supplier/template.njk [hunk 1]
```diff
@@ -49,13 +53,28 @@
...
           <div data-test-id="supplier-address-country" class="nhsuk-u-font-size-19 nhsuk-u-margin-bottom-5">{{ supplierData.address.country }}</div>
           <input type="hidden" name="country" value="{{ supplierData.address.country }}" />
+        </div>
+      </div>
+
+      <hr class="bc-c-order-table--hr"/>
+
+      <div class="nhsuk-grid-row">
+        <div data-test-id="primary-contact-fields" class="nhsuk-grid-column-two-thirds">
+          <h3 data-test-id="primary-contact-heading" class="nhsuk-heading-s nhsuk-u-margin-bottom-3">{{ primaryContactHeading }}</h3>
```
Comment (kat1906) [inline app/pages/sections/supplier/supplier/template.njk:63]: I made this bigger to be more in line with the designs and also h2 (see my PR)

**Comment by kat1906** [comment]
tech debt? 😄

**Comment by kat1906** [comment]
> you should definitely extract this out as it is now in 2 places

Tech debt? 😄

**Comment by alibombaye** [comment]
> > you should definitely extract this out as it is now in 2 places
> 
> Tech debt? 😄

haha yes, I think if we have to repeat this once more then it's definitely with extracting out. But think this is the last case of it.. I'll take a look at future stories to see if another one of these is coming up.

**Pull request merged**